ipykernel/kernelbase.py

```diff
@@ -880,18 +880,23 @@ async def connect_request(self, socket, ident, parent):
 
     @property
     def kernel_info(self):
-        info = {
+        from .debugger import _is_debugpy_available
+
+        supported_features: list[str] = []
+        if self._supports_kernel_subshells:
+            supported_features.append("kernel subshells")
```

Comment:
Is it necessary to have "kernel", or could we just put "subshells"?

Reply:
"kernel subshells" is what is says in the JEP (https://github.com/jupyter/enhancement-proposals/blob/master/kernel-subshells/kernel-subshells.md#identify-optional-feature) so that is what it must be.